### PR TITLE
clear local storage on logout and add reset func in column selection

### PR DESCRIPTION
**Pull request by raylrui**
1. add local storage clear function when use logout out. 
2. add reset button in display column selections.

### CODE REVIEW

File: src/components/workflows/WorkflowTable.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ import { useLocalStorage } from "../utils/LocalStorage";
 export default function CustomTable(props) {
   const { items, paginationProps, handleChangeQuery, ordering } = props;
 
+  console.log("items", items);
```
Comment (williamputraintan) [inline src/components/workflows/WorkflowTable.js:30]: I think we could leave this out 🙂

**Review by williamputraintan — APPROVED** [review]
LGTM!

File: src/components/workflows/WorkflowTable.js (hunk 1 above)
Comment (raylrui) [reply src/components/workflows/WorkflowTable.js:30]: will remove it before merge. Thanks

Comment (victorskl) [reply src/components/workflows/WorkflowTable.js:30]: good stuff!

**Pull request merged**